### PR TITLE
Adds a version of tator lings where anyone can be a ling/tator for admemes

### PR DESCRIPTION
**Pull request by fluffe9911**
🆑 
add: new subtype of tator lings where anyone can be a tator/ling for admemery!
/🆑

### CODE REVIEW

**Comment by ynot01** [comment]
even like captain and sec?

**Comment by fluffe9911** [comment]
yes

**Comment by boodaliboo** [comment]
Time to give captain some job exclusives

**Comment by alexkar598** [comment]
Admins can varedit ok

**Comment by fluffe9911** [comment]
yes but this makes it more convenient for the average admeme and is only like 5 lines of code

**Comment by Hopekz** [comment]
Quality of life PR.
+1

**Review by alexkar598 — CHANGES_REQUESTED** [review]
Make it a flag to ignore the check that can be activated from secrets panel instead. E.

File: code/game/gamemodes/changeling/traitor_chan.dm [hunk 1]
```diff
@@ -81,3 +81,8 @@
 	return "The Syndicate has started some experimental research regarding humanoid shapeshifting.  There are rumors that this technology will be field tested on a Nanotrasen station \
 			for infiltration purposes.  Be advised that support personel may also be deployed to defend these shapeshifters. Trust nobody - suspect everybody. Do not announce this to the crew, \
 			as paranoia may spread and inhibit workplace efficiency."
+
+/datum/game_mode/traitor/changeling/trustnoone
+	name = "traitor + lings + anybody can be a antag"
```
Comment (Ktlwjec) [inline code/game/gamemodes/changeling/traitor_chan.dm]: name = "traitor + lings + anybody can be an antag"

**Review by Ktlwjec — CHANGES_REQUESTED** [review]

**Comment by AsV9** [comment]
> Make it a flag to ignore the check that can be activated from secrets panel instead. E.

It's fine to leave this as a gamemode like raginmages.

**Pull request merged**